Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Cart.kt

```diff
@@ -16,20 +16,329 @@
+        SnackImage(
+            imageUrl = snack.imageUrl,
+            modifier = Modifier
+                .preferredSize(100.dp)
+                .constrainAs(image) {
+                    top.linkTo(parent.top, margin = 16.dp)
+                    bottom.linkTo(parent.bottom, margin = 16.dp)
+                    start.linkTo(parent.start)
+                }
+        )
+        Text(
+            text = snack.name,
+            style = MaterialTheme.typography.subtitle1,
+            color = JetsnackTheme.colors.textSecondary,
+            modifier = Modifier.constrainAs(name) {
+                linkTo(
+                    start = image.end,
+                    startMargin = 16.dp,
+                    end = remove.start,
+                    endMargin = 16.dp,
+                    bias = 0f
+                )
+            }
+        )
+        IconButton(
+            onClick = { /* todo */ },
+            modifier = Modifier.constrainAs(remove) {
+                top.linkTo(parent.top)
+                end.linkTo(parent.end)
+            }.padding(top = 12.dp)
+        ) {
+            Icon(asset = Icons.Filled.Close, tint = JetsnackTheme.colors.brand)
+        }
+        Text(
+            text = snack.tagline,
+            style = MaterialTheme.typography.body1,
+            color = JetsnackTheme.colors.textHelp,
+            modifier = Modifier.constrainAs(tag) {
+                linkTo(
+                    start = image.end,
+                    startMargin = 16.dp,
+                    end = parent.end,
+                    endMargin = 16.dp,
+                    bias = 0f
+                )
+            }
+        )
+        Spacer(
+            Modifier
+                .preferredHeight(8.dp)
+                .constrainAs(priceSpacer) {
+                    linkTo(top = tag.bottom, bottom = price.top)
+                }
+        )
+        Text(
+            text = "$12.99",
+            style = MaterialTheme.typography.subtitle1,
+            color = JetsnackTheme.colors.textPrimary,
+            modifier = Modifier.constrainAs(price) {
+                linkTo(
+                    start = image.end,
+                    end = quantity.start,
+                    startMargin = 16.dp,
+                    endMargin = 16.dp,
+                    bias = 0f
+                )
+            }
+        )
+        QuantitySelector(
+            count = count,
+            decreaseItemCount = { if (count > 0) updateCount(count - 1) },
+            increaseItemCount = { updateCount(count + 1) },
+            modifier = Modifier.constrainAs(quantity) {
+                baseline.linkTo(price.baseline)
+                end.linkTo(parent.end)
+            }
+        )
+        JetsnackDivider(
+            Modifier.constrainAs(divider) {
+                linkTo(start = parent.start, end = parent.end)
+                top.linkTo(parent.bottom)
+            }
+        )
+    }
+}
+
+@Composable
+fun SummaryItem(modifier: Modifier = Modifier) {
```

Comment:
Maybe add a TODO comment to hoist state here? In that way, people know this is not a good practice and that changes are coming to this composable